src/appleseed/foundation/math/phasefunction.cpp

```diff
@@ -128,4 +128,38 @@ float IsotropicPhaseFunction::sample(const Vector3f& outgoing, const Vector2f& s
     return RcpFourPi<float>();
 }
 
+
+//
+// DwivediPhaseFunction class implementation.
+//
+
+DwivediPhaseFunction::DwivediPhaseFunction(const float v)
+    : m_v(v)
+{
+}
+
+float DwivediPhaseFunction::sample(const Vector3f& outgoing, const Vector2f& s, Vector3f& incoming) const
+{
+    const float cosine = m_v - sample_rcp_distribution(s[0], m_v - 1.0f, m_v + 1.0f);
+    const float sine = std::sqrt(saturate(1.0f - cosine * cosine));
```

Comment:
You can remove all `std::` qualifiers since you're already importing the `std` namespace.